Index: testing/test_main.py
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>import pytest\r\nfrom web_client.main import *\r\nfrom web_client.cpu import Cpu\r\nfrom web_client.gpu import Gpu\r\nfrom web_client.ram import Ram\r\nfrom web_service_project.common_route_methods.parse import parse_query_string\r\n\r\n\r\ndef test_home_message():\r\n    result = home_message()\r\n    assert isinstance(result, dict)\r\n
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/testing/test_main.py b/testing/test_main.py
--- a/testing/test_main.py	(revision 45d17c84f0701673fbb09954c9887cc447abf8df)
+++ b/testing/test_main.py	(date 1748596413973)
@@ -9,3 +9,10 @@
 def test_home_message():
     result = home_message()
     assert isinstance(result, dict)
+
+
+def test_get_component_names():
+    result = get_component_names()
+    assert isinstance(result, list)
+
+
Index: ../.idea/modules.xml
===================================================================
diff --git a/../.idea/modules.xml b/../.idea/modules.xml
deleted file mode 100644
--- a/../.idea/modules.xml	(revision 45d17c84f0701673fbb09954c9887cc447abf8df)
+++ /dev/null	(revision 45d17c84f0701673fbb09954c9887cc447abf8df)
@@ -1,8 +0,0 @@
-<?xml version="1.0" encoding="UTF-8"?>
-<project version="4">
-  <component name="ProjectModuleManager">
-    <modules>
-      <module fileurl="file://$PROJECT_DIR$/.idea/web_service.iml" filepath="$PROJECT_DIR$/.idea/web_service.iml" />
-    </modules>
-  </component>
-</project>
\ No newline at end of file
